docs/quickstart.rst

```diff
@@ -8,51 +8,156 @@ up pg_auto_failover to replicate data between them. We’ll simulate failure in
+
+.. code-block:: bash
+
+  # run this in your local shell as well
+
+  vm_ip () {
+    az vm list-ip-addresses -g ha-demo -n ha-demo-$1 -o tsv \
+      --query '[] [] .virtualMachine.network.publicIpAddresses[0].ipAddress'
+  }
 
 .. _quickstart_install:
 
 Install the "pg_autoctl" executable
---------------------------------
+-----------------------------------
 
-This guide uses Red Hat Linux, but similar steps will work on other distributions. All that differs are the packages and paths. See :ref:`install`.
+This guide uses Debian Linux, but similar steps will work on other
+distributions. All that differs are the packages and paths. See :ref:`install`.
 
-pg_auto_failover is distributed as a single binary with subcommands to
-initialize and manage a replicated PostgreSQL service. We’ll install the
-binary with the operating system package manager.
+The pg_auto_failover system is distributed as a single ``pg_autoctl`` binary
+with subcommands to initialize and manage a replicated PostgreSQL service.
+We’ll install the binary with the operating system package manager on all
+nodes. It will help us run and observe PostgreSQL.
 
 .. code-block:: bash
 
-  curl https://install.citusdata.com/community/rpm.sh | sudo bash
-
-  sudo yum install -y pg-auto-failover10_11
+  for node in monitor a b app
+  do
+  az vm run-command invoke \
+     --resource-group ha-demo \
+     --name ha-demo-${node} \
+     --command-id RunShellScript \
+     --scripts \
+        "curl https://install.citusdata.com/community/deb.sh | sudo bash" \
+        "sudo apt-get install -q -y postgresql-common" \
+        "echo 'create_main_cluster = false' | sudo tee -a /etc/postgresql-common/createcluster.conf" \
+        "sudo apt-get install -q -y postgresql-11-auto-failover-1.2" \
+        "sudo usermod -a -G postgres ha-admin" &
+  done
+  wait
 
 .. _quickstart_run_monitor:
 
 Run a monitor
 -------------
 
-The pg_auto_failover monitor is the first component to run. It periodically attempts
-to contact the other nodes and watches their health. It also maintains
-global state that “keepers” on each node consult to determine their own
-roles in the system.
-
-Because we’re running everything on a single machine, we give each
-PostgreSQL instance a separate TCP port. We’ll give the monitor a
-distinctive port (6000):
+The pg_auto_failover monitor is the first component to run. It periodically
+attempts to contact the other nodes and watches their health. It also
+maintains global state that “keepers” on each node consult to determine their
+own roles in the system.
 
 .. code-block:: bash
 
-   sudo su - postgres
-   export PATH="/usr/pgsql-11/bin:$PATH"
+   # on the monitor virtual machine
 
-   pg_autoctl create monitor   \
-     --pgdata ./monitor     \
-     --pgport 6000
+   ssh -l ha-admin `vm_ip monitor` -- \
+     pg_autoctl create monitor \
+       --auth trust \
+       --ssl-self-signed \
+       --pgdata monitor \
+       --pgctl /usr/lib/postgresql/11/bin/pg_ctl
```

Comment:
For now it times out, and I checked and reinstalled the network rules and it seems good...

```
~ ssh -l ha-admin `vm_ip monitor`
ssh: connect to host 137.135.120.211 port 22: Operation timed out
```